src/templates/timeslider.html

```diff
@@ -3,7 +3,7 @@
     , langs = require("ep_etherpad-lite/node/hooks/i18n").availableLangs
 %>
 <!doctype html>
-<html class="<%=settings.skinVariants%>">
+<html class="timeslider <%=settings.skinVariants%>">
```

Comment:
If we use `pad` instead then we can reuse the existing CSS rules:

```suggestion
<html class="pad <%=settings.skinVariants%>">
```